test/lib/rails-jquery-autocomplete/autocomplete_test.rb

```diff
@@ -65,6 +66,7 @@ class AutocompleteTest < Test::Unit::TestCase
       should 'return an instance of HashWithIndifferentAccess' do
         item = mock(Object)
         mock(item).send(:name).times(2) { 'Object Name' }
+        stub(item).kind_of?(Hash){ false }
```

Comment:
Prefer `Object#is_a?` over `Object#kind_of?`.<br>Space missing to the left of {.